### ISSUE TITLE
Add manifest.json

### ISSUE THREAD

**Issue by k1rd3rf** [k1rd3rf/min_renovasjon#1]

**Comment by ezpl** [comment]
Yup, HACS reports "No manifest.json file found 'custom_components/min_renovasjon/manifest.json'" when trying to add custom repository
